apps/api/src/server/migration/migrators/folder/folder-name-map.js

```diff
@@ -236,6 +235,7 @@ const traversePartialMap = (
 	}
 	if (typeof currentLevelObject === 'string') {
 		cbosPathObject.cbosPathParts.push(currentLevelObject);
+		nextLevel ? cbosPathObject.failedToMatchParts.push(nextLevel) : null;
```

Comment:
Curious on why you used a ternary here, it looks like it would work but the null at the end seems odd to me since we are not doing anything if nextLevel is falsy. If we are avoiding an if statement we could do `&&` and do `nextLevel && cbosPathObject.failedToMatchParts.push(nextLevel)` although I'd say if statement is more explicit. No need to change anything, just wondering.

Reply:
good spot, that would be nicer, have changed in recent commit